backend/open_webui/routers/audio.py

```
@@ -287,33 +354,39 @@ async def speech(request: Request, user=Depends(get_verified_user)):
 
             return FileResponse(file_path)
 
         except Exception as e:
             log.exception(e)
             detail = None
 
-            try:
-                if r.status != 200:
-                    res = await r.json()
+            status_code = 500
+            detail = f"Open WebUI: Server Connection Error"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)